src/Infrastructure/Persistence/Repository/UnitOfWork.cs

```diff
@@ -6,7 +6,9 @@ namespace Persistence.Repository;
 public class UnitOfWork : IUnitOfWork
 {
     private readonly SocialSyncDbContext _dbContext;
-    private FollowsRepository? _followsRepository;
+    private IFollowsRepository? _followsRepository;
```

Comment:
The Generic Repo must be replaced with the Implementatoin

Reply:
I have changed the interfaces with the actual classes